editions/tw5.com/tiddlers/WebServer Parameter_ server-settings.tid

```diff
@@ -0,0 +1,29 @@
+caption: server-settings
+created: 20211111003224763
+creator: Joshua Fontany
+modified: 20211111023427713
+modifier: Joshua Fontany
+tags: [[WebServer Parameters]]
+title: WebServer Parameter: server-settings
+type: text/vnd.tiddlywiki
+
+The [[web server configuration parameter|WebServer Parameters]] ''server-settings'' contains the pathname of a [[JSON file|https://en.wikipedia.org/wiki/JSON]] containing attribute-value pairs for //all other// [[web server configuration parameter|WebServer Parameters]].
```

Comment:
Could we store this data in the tiddlywiki.info file?

Reply:
@Jermolene This was brought up below by @saqimtiaz. While I like how this simplifies things for a single-wiki-server setup, I think it would be difficult in certain more complex cases (for example, I run a authenticated wiki server for my own multiple project wikis, and am often re-launching with the same server settings but pointing to a different wiki... but I also use a `writers=(anon)` wiki server whenever I update TW documentation). I would want point to a file outside of TW repository for cases where I am launching an edition from within the TW repository...

Hm, it would definitely give users not familiar with Node.js servers a "central settings file" to reduce cognitive load. That's a big point in favor. On the other hand, you could do `server-settings=./tiddlywiki.info` from the command line and achieve the same thing. This could be called out in the documentation. Thoughts?